### DISCUSSION TITLE
Stat Panel: Oldest data point used?

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64461] [Dashboards, Panels & Visualizations]
When using Stat panels, I am used to having this at the end of my psql query so that I only get the last data point.
```
...
order by time desc
limit 1
```
But if I have a Stat panel for which I want to set the graph mode to "Area", and remove the `limit 1` from the query, the graph looks good but the value that is printed is the value associated with the earliest time stamp, not the last timestamp as I expect. I can change `order by time desc` to `order by time asc` to work around this, but seems to me like it should always be printing the last value regardless? Setting a calculation of `Last *` doesn't seem to make a difference.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82832. Feel free to continue the discussion around this there. Thank you!
